internal/command/test.go

```diff
@@ -976,6 +976,9 @@ func (runner *TestFileRunner) Cleanup(file *moduletest.File) {
 		}
 		runner.Suite.View.DestroySummary(diags, state.Run, file, updated)
 
+		if updated.HasManagedResourceInstanceObjects() {
+			views.SaveErroredTestStateFile(updated, state.Run, file, runner.Suite.View)
```

Comment:
I see your point. But technically the `views` package should only do view related things.
Ideally, `views` package should implement a way to print out "I'm going to save the error state file", and to dump the error state file to screen / JSON.

Then, it would be this package's responsibility to: 1. Call `views` to print out "I'm going to save the error state file". 2. Attempt to write the error state file. 3. If write failed, call `views` to dump the error state file

However, this is a nit. Feel free to keep it as is, and resolve the comment